docs/Changelog.md

```diff
@@ -34,6 +34,8 @@
 * [#5465](https://github.com/TouK/nussknacker/pull/5465) Fix: Wasn't possible to compare a Record with a Map.
 * [#5457](https://github.com/TouK/nussknacker/pull/5457) Fix: Array types wasn't serialized correctly which caused deserialization error during node validation.
 * [#5475](https://github.com/TouK/nussknacker/pull/5475) SpEL expressions checking improvement: The equals operator used with two Lists with different element types is reported as an error
+* [#5438](https://github.com/TouK/nussknacker/pull/5438) [#5495](https://github.com/TouK/nussknacker/pull/5495) Alignment in the api of primary (deploy/cancel) actions and the experimental api of custom actions.
```

Comment:
Let's keep this on the level, that will be understandable for an average Nussknacker's, not too advanced user. See our checklist in PR. I suggest skipping the part with value class and starting from some high level concept. I think that "Improvement in DeploymentManager API: Alignment ..." would be more understandable for the user. WDYT?

Reply:
Fixed